pkgs/development/python-modules/sqlalchemy/default.nix

```
@@ -30,14 +27,17 @@ buildPythonPackage rec {
 
   pytestFlagsArray = [ "-n auto" ];
 
+  propagatedBuildInputs = lib.optional isPy3k greenlet
```

```suggestion
  propagatedBuildInputs = [ greenlet ]
```
Only ``py3.X`` is supported (by disable), so the optional is not needed.